.circleci/config.yml

```diff
@@ -414,6 +414,9 @@ commands:
             export PARTITION_TOTAL=${CIRCLE_NODE_TOTAL}
             export PARTITION_ID=${CIRCLE_NODE_INDEX}
             export PARALLEL_FLAG="-p 1"
+            # TODO: the next 2 are temporary
+            export E2E_TEST_FILTER=SCRIPTS 
+            export AWS_NET_UPLOAD=znet_done.tar.bz2
```

Comment:
But should be merged to feature branch?

Reply:
no, they should not. I'll open a new branch shortly that contains the experimental changes.